url-updater/src/main/java/com/devonfw/tools/ide/url/updater/AbstractUrlUpdater.java

```diff
@@ -363,8 +364,7 @@ private boolean isValidDownload(String url, String toolWithEdition, String versi
   }
 
   /**
-   * Checks if the content type was not of type text (this method is required because {@link com.devonfw.tools.ide.tool.pip.PipUrlUpdater} returns text and
-   * needs to be overridden)
+   * Checks if the content type was not of type text (this method is required because {@link PipUrlUpdater} returns text and needs to be overridden)
```

Comment:
```suggestion
   * Checks if the content type was not of type text (this method is required because {@link com.devonfw.tools.ide.url.tool.pip.PipUrlUpdater} returns text and
   * needs to be overridden).
```

Reply:
Replaced with FQR now.